### ISSUE TITLE
error on POST to notificationcuecustoms

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispersservices_django#341]
![image](https://user-images.githubusercontent.com/6978380/75804640-f0a9ae00-5d45-11ea-8c9e-660d5deb8470.png)

note this does include the field `event_affected_count_operator` which is currently in dispute - but same result when that is omitted. 

JSON:
```
{
  "event": null,
  "event_affected_count": 20,
  "event_affected_count_operator": "__gte",
  "event_location_administrative_level_one": {
    "operator": "OR",
    "values": [
      51,
      27
    ]
  },
  "species": {
    "operator": "OR",
    "values": [
      748
    ]
  },
  "species_diagnosis_diagnosis": {
    "operator": "OR",
    "values": [
      686
    ]
  },
  "event_location_land_ownership": {
    "operator": "OR",
    "values": [
      3
    ]
  },
  "new_notification_cue_preference": {
    "create_when_new": true,
    "create_when_modified": true,
    "send_email": false
  }
}
```

```
Traceback:

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/viewsets.py" in view
  114.             return self.dispatch(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  505.             response = self.handle_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in handle_exception
  465.             self.raise_uncaught_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in raise_uncaught_exception
  476.         raise exc

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  502.             response = handler(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/mixins.py" in create
  18.         serializer.is_valid(raise_exception=True)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/serializers.py" in is_valid
  234.                 self._validated_data = self.run_validation(self.initial_data)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/serializers.py" in run_validation
  436.             value = self.validate(value)

File "/var/www/whispersservices_django/whispersservices/serializers.py" in validate
  4103.                             or data[field]['operator' not in ("AND", "OR")])):

```

**Comment by aaronstephenson** [comment]
Looks like I was missing the closing bracket on line 4103 of serializers.py.

`event_affected_count_operator` is ignored, since it isn't registered as a field on that serializer.

**Comment by aaronstephenson** [comment]
Fixed in 146b438e
